app/descriptor/views.py

```diff
@@ -327,3 +339,84 @@ def delete_descriptor(desc_id):
             return render_template('descriptor/manage_descriptor.html',
                                    desc=descriptor, is_option=is_option)
     return redirect(url_for('descriptor.index'))
+
+@descriptor.route('/change-required-option-descriptor', methods=['GET', 'POST'])
+@login_required
+def change_required_option_descriptor():
+    descriptors = Descriptor.query.all()
+    choices = []
+    for d in descriptors:
+        if d.values:
+            choices.append((d.name, d.name))
+    req_opt_desc = RequiredOptionDescriptor.query.all()[0]
+    current_name = ""
+    if req_opt_desc.descriptor_id != -1:
+        descriptor = Descriptor.query.filter_by(
+            id=req_opt_desc.descriptor_id
+        ).first()
+        if descriptor is not None:
+            current_name = descriptor.name
+    if current_name != "":
+        setattr(ChangeRequiredOptionDescriptorForm, 'descriptor', SelectField('Option Descriptor', choices=choices, validators=[InputRequired()], default=current_name))
```

Comment:
Because that is the current required option descriptor? That makes the most sense to me. This form is for changing which descriptor is the required option descriptor.